frontend/server/config.default.php

```diff
@@ -17,8 +17,14 @@
 # ####################################
 try_define('IS_TEST', false);
 try_define('OMEGAUP_ENABLE_SOCIAL_MEDIA_RESOURCES', true);
-try_define('OMEGAUP_TEST_ROOT', OMEGAUP_ROOT . '/tests/controllers/');
-try_define('OMEGAUP_TEST_RESOURCES_ROOT', OMEGAUP_ROOT . '/tests/resources/');
+try_define(
+    'OMEGAUP_TEST_ROOT',
+    sprintf(strval(OMEGAUP_ROOT), '/tests/controllers/')
```

Comment:
```suggestion
    sprintf('%s/tests/controllers/', strval(OMEGAUP_ROOT))
```

igual en todos los demás lugares.

Reply:
Listo